tests/main.nf.test

```diff
@@ -0,0 +1,39 @@
+nextflow_pipeline {
+
+    name "Test fastqrepair pipeline"
+    script "../main.nf"
+
+    tag "pipeline"
+    tag "fastqrepair"
+
+    test("30 paired-end reads, two FASTQ are empty") {
+
+        when {
+            params {
+                input                   = 'https://raw.githubusercontent.com/nf-core/test-datasets/fastqrepair/testdata/samplesheet_30reads.csv'
+                outdir                  = "$outputDir/30reads"
+
+                publish_all_tools       = true
+                num_splits              = 2
+            }
+        }
+
+        then {
+            assertAll(
+                { assert workflow.success },
+                { assert snapshot(
+                    path("$outputDir/30reads/repaired").list(),
+                    path("$outputDir/30reads/repaired/test_30reads_2.report"),
+                    path("$outputDir/30reads/pipeline_info/nf-core_fastqrepair_versions.yml"),
```

Comment:
```suggestion
                    path("$outputDir/30reads/pipeline_info/nf-core_fastqrepair_versions.yml"),
                    file("$outputDir/30reads/QC/multiqc/multiqc_report.html").exists(),
                    file("$outputDir/30reads/QC/fastqc/<FILENAMEHERE>.html").exists(),
```

My point from before still stands, I don't know if the exact code in the suggestion works but you get the idea (note the HTMLs are variable thus the .exists()`)